cephfs/file_test.go

```diff
@@ -405,3 +405,63 @@ func TestFchown(t *testing.T) {
 	assert.Equal(t, uint32(stats.Sys().(*syscall.Stat_t).Uid), bob)
 	assert.Equal(t, uint32(stats.Sys().(*syscall.Stat_t).Gid), bob)
 }
+
+func TestFstatx(t *testing.T) {
+	fname := "test_fstatx.txt"
+
+	mount := fsConnect(t)
+	defer fsDisconnect(t, mount)
+
+	f, err := mount.Open(fname, os.O_RDWR|os.O_CREATE, 0600)
+	assert.NoError(t, err)
+	assert.NotNil(t, f)
+	assert.NoError(t, f.Close())
+	defer func() { assert.NoError(t, mount.Unlink(fname)) }()
+
+	t.Run("emptyFile", func(t *testing.T) {
+		f, err := mount.Open(fname, os.O_RDWR, 0600)
+		assert.NoError(t, err)
+		assert.NotNil(t, f)
+
+		st, err := f.Fstatx(StatxBasicStats, 0)
+		assert.NoError(t, err)
+		assert.NotNil(t, f)
+
+		assert.Equal(t, uint16(0600), st.Mode&0600)
+		assert.Equal(t, uint64(0), st.Size)
+	})
+
+	t.Run("populateFile", func(t *testing.T) {
+		f, err := mount.Open(fname, os.O_RDWR|os.O_CREATE, 0600)
```

Comment:
the file is already created, no need for `O_CREATE`, but it does not hurt either

Reply:
Sure. If not a literal copy-n-paste an artifact of habit based on other lines in the test. I'll drop it if I make any other changes to the PR.